clients/pkg/promtail/targets/gcplog/pull_target.go

```diff
@@ -1,17 +1,18 @@
 package gcplog
 
 import (
-	"cloud.google.com/go/pubsub"
 	"context"
+	"io"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/pubsub"
```

Comment:
nit: in general can we avoid adding changes like this in a feature PR, can make it harder to review

if there's a lint failure that made this necessary that's fine, otherwise it's easier to review features and whitespace/cleanup changes separately

Reply:
Uhh my bad, I've been running the `Makefiles` linter to avoid CI back and forth, and it seems it fixes some import orders if it founds them. I'll revert those changes!